### DISCUSSION TITLE
Video doesn't download

### DISCUSSION THREAD

**Discussion by ballo** [aclap-dev/video-downloadhelper#1298] [Help and Q&A]
### Your question

https://video.storyful.com/record/32824

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"unset":true}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "used_history_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 11,
    "successfull_dl": 12
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.4169, Firefox 130.0.1 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

When I visited your web page, without launching playback, VDH told me this:

![#01](https://github.com/user-attachments/assets/920e7b0f-ef89-43de-9a53-40478d4d8ea5)

The indicated variant download just fine.  This is what the file looked like:

![#02](https://github.com/user-attachments/assets/b6761502-0359-459a-ba43-507492c1d89e)

It played just fine in VLC.

I don't know what the problem might be.  There's 2 big differences between your environment & mine.  First off, I'm running Windows & you're on Mac.  Second, I'm using Firefox & you're using Chrome.  I suspect it's the difference between Firefox & Chrome that is causing the problem here.  It wouldn't be the first time that Firefox handled a page better than Chrome.  If you're willing to experiment, you might install Firefox, then install the latest VDH beta for Firefox.  See if it downloads your video for you.

You might also look in the Chrome Developer Tools in the Network Monitor.  For me, it showed no HLS master manifest but it did show an HLS stream manifest for this content.  That is reflected in my image of the VDH menu as the RawHLS variant.  All that means is this video is not available in multiple resolutions.  There's just the one resolution & VDH had no problem with it for me in Firefox.  The HLS stream manifest is identified in the Network Monitor as having a Type of vnd.apple.mpegurl.

![#03](https://github.com/user-attachments/assets/6b9703d5-d7a3-4364-b49a-b7291367ee32)

You might try this trick:

1. Double click the HLS stream manifest in the Network Monitor.
2. That will open a new browser tab.
3. If it works anything like Firefox, it will give you a prompt to either choose an application for opening the manifest or download it.
4. Do neither.  Just dismiss the dialog.
5. See whether VDH has recognized anything.  You might be able to download the video from that.

Let's hear from you how you get on with this.

**Reply by GrampaWildWilly** [reply]
I should mention that after you double click that entry in the Network Monitor, you'll think it didn't do anything.  But it did.  Switch tasks back to your original video window.  The new tab with be there.
